eng/validation/build-arcadewithrepo.ps1

```diff
@@ -39,34 +39,7 @@ $global:subscribedBranchName = $subscribedBranchName
 Write-Host "##vso[task.setvariable variable=arcadeVersion;isOutput=true]${global:arcadeSdkVersion}"
 Write-Host "##vso[task.setvariable variable=qualifiedRepoName;isOutput=true]${global:githubOrg}/${global:githubRepoName}"
 
-# Get a temporary directory for a test root. Use the agent work folder if running under azdo, use the temp path if not.
-$testRootBase = if ($env:AGENT_WORKFOLDER) { $env:AGENT_WORKFOLDER } else { $([System.IO.Path]::GetTempPath()) }
-$testRoot = Join-Path -Path $testRootBase -ChildPath $([System.IO.Path]::GetRandomFileName())
-New-Item -Path $testRoot -ItemType Directory | Out-Null
-
-function Get-LatestBuildSha()
-{
-    ## Verified that this API gets completed builds, not in progress builds
-    $headers = Get-AzDOHeaders
-    $uri = "https://dev.azure.com/${global:azdoOrg}/${global:azdoProject}/_apis/build/latest/${global:buildDefinitionId}?branchName=${global:subscribedBranchName}&api-version=5.1-preview.1"
-    $response = (Invoke-WebRequest -Uri $uri -Headers $headers -Method Get) | ConvertFrom-Json
-
-    ## Report non-green repos for investigation purposes. 
-    if(($response.result -ne "succeeded") -and ($response.result -ne "partiallySucceeded"))
-    {
-        Write-Host "##vso[task.setvariable variable=buildStatus;isOutput=true]NoLKG"
-        Write-Warning "The latest build on '${global:subscribedBranchName}' branch for the '${global:githubRepoName}' repository was not successful."
-    }
-
-    if("" -eq $response.triggerInfo)
-    {
-        return $response.sourceVersion
-    }
-    else 
-    {
-        return $response.triggerInfo.'ci.sourceSha'
-    }
-}
+..\validation-functions.ps1
```

Comment:
I think this should be
```suggestion
.\validation-functions.ps1
```

Since validation-functions.ps1 lives in the same folder as this.

Reply:
the first `.` is the dot-source operator, saying, please bring this script in. The second part is the path to the script you're bringing in

Reply:
You also need to . it.
```suggestion
. .\validation-functions.ps1
```

Reply:
So it should be
.  .\validation-functions.ps1
?

Reply:
Both suggestions are correct. 

You need <dot><space><filepath>

so 
`. .\validation-functions.ps1`